File: test/binaryTest.php
<?php

require_once 'app/binary.php';

class binaryTest extends PHPUnit_Framework_TestCase
{
    function testToString() {
        $this->assertEquals("0", toString(0));
        $this->assertEquals("1", toString(1));
        $this->assertEquals("10", toString(2));
        $this->assertEquals("11", toString(3));
        $this->assertEquals("100", toString(4));
    }
}

File: binary_test.go
package binary

import "testing"

func TestToString(t *testing.T) {
	if ToString(0) != "0" {
		t.Errorf("found '%v' expected '0'", ToString(0))
	}
	if ToString(1) != "1" {
		t.Errorf("found '%v' expected '1'", ToString(1))
	}
	if ToString(2) != "10" {
		t.Errorf("found '%v' expected '10'", ToString(2))
	}
	if ToString(3) != "11" {
		t.Errorf("found '%v' expected '11'", ToString(3))
	}
	if ToString(4) != "100" {
		t.Errorf("found '%v' expected '100'", ToString(4))
	}
}